e2e/src/api/specs/asset.e2e-spec.ts

```diff
@@ -1222,7 +1222,7 @@ describe('/asset', () => {
       },
     ];
 
-    it(`should upload and generate a thumbnail for different file types`, async () => {
+    it(`should upload and generate a thumbnail for different file types`, { timeout: 60_000 }, async () => {
```

Comment:
The timeout changes should be unnecessary since the setting is disabled by default, right?